docs/plugin-development/reverse-engineering/index.md

```diff
@@ -22,7 +22,7 @@ useful information present in it. To this end, the community has created and
 maintained a project known as
 [FFXIVClientStructs](https://github.com/aers/FFXIVClientStructs), which provides
 a general source of information on the game internals and provides set of C#
-bindings that effectively allow plugins to use the game as a library. However,
+bindings that effectively allow plugins to use the game as a library. Docs can be found [here](https://ffxiv.wildwolf.dev) however,
```

Comment:
Two things:

- This would be better grammatically as "here, however".
- Is this docs site the official one for the project? I'm not a big fan of linking to external sites that may be unreliable (see: all the times my home server went down with Loggy lol)

Reply:
The domain will be the doc link for CS yes. Currently it is hosted on my own server behind Cloudflare proxy, but might look into getting it to GitHub pages at some point.